lib/matplotlib/tests/test_table.py

```diff
@@ -229,3 +233,34 @@ def test_table_bbox(fig_test, fig_ref):
                   loc='center',
                   bbox=Bbox.from_extents(0.1, 0.2, 0.9, 0.8)
                   )
+
+
+@check_figures_equal(extensions=['.png'])
```

Comment:
There shouldn't be a dot here
```suggestion
@check_figures_equal(extensions=['png'])
```